drivers/clock_control/clock_control_nrf2_fll16m.c

```diff
@@ -225,27 +221,6 @@ static int api_get_rate_fll16m(const struct device *dev,
 static int fll16m_init(const struct device *dev)
 {
 	struct fll16m_dev_data *dev_data = dev->data;
-	nrf_bicr_lfosc_mode_t lfosc_mode;
```

Comment:
question: should we update BICR schema to remove this mode?

Reply:
It is no longer used, so probably yes :)